index.bs

```diff
@@ -5362,3 +5533,13 @@ Possible mitigations:
     presence from the reporting origin. Compared to the previous mitigation, the
     proxy server could itself handle the [=event-level report/trigger priority=]
     functionality, at the cost of increased complexity in the proxy.
+
+## Attribution scope ## {#attribution-scope}
+
+*This section is non-normative.*
+
+It is possible for an adversary to register multiple [=source type/navigation=] sources in response to a single navigation, and use these multiple sources, each with a different [=attribution source/attribution scopes=] value, to gain additional information about a user based on which attribution scope is chosen. To prevent this abuse the number of unique attribution scope sets per reporting origin per navigation needs to be limited.
```

Comment:
Could you please clarify whether an empty attribution scopes set would be considered for this limit? Is there any privacy concern on having both empty and non-empty attribution scopes set per navigation?

Reply:
Thought it again, as empty vs non-empty attribution scopes set is handled in source registration, it's probably fine. But maybe we should clarify we only consider non-empty set here.